src/Profile/UIProfile.cpp

```diff
@@ -130,7 +130,7 @@ Profile::Load(const ProfileMap &map, UISettings &settings)
   if (settings.custom_dpi < 120 || settings.custom_dpi > 520)
     settings.custom_dpi = 0;
 
-  map.Get(ProfileKeys::EnableTAGauge, settings.enable_thermal_assistant_gauge);
+  map.GetEnum(ProfileKeys::TAPosition, settings.thermal_assistant_position);
```

Comment:
This loses existing configuration. If you refactor settings, you should migrate old configuration data.